test/test_fx_experimental.py

```diff
@@ -584,6 +584,21 @@ def test_conv_bn_fusion(self):
 
         self.assertEqual(fused(inp), rn18(inp))
 
+    def test_rewrite_traced_module(self):
```

Comment:
What new code path is this testing?

Reply:
This is testing tracing a GraphModule with RewritingTracer.

Reply:
I might be wrong, but I thought this was already covered by the `test_call_to_assert_...` tests in this file

Reply:
Oh this test case is testing the edge case where we symbolic trace a module first then use RewritingTracer to trace this already traced module while `test_call_to_assert_...` are all calling RewritingTracer on nn.Module. The code path this test case go through is this branch
```
if len(new_keys) == 0:
    fn_compiled = globals_dict["forward"]
```

Reply:
I see, sorry I missed that!

Reply:
Totally fine, thanks for reviewing it! Let me know if any other changes are needed.